editor/src/views/Crashes/CrashChangeLog.js

```diff
@@ -63,57 +71,79 @@ const isNewRecordEvent = change => change.operation_type === "create";
 
 /**
  * The primary UI component which renders the change log with clickable rows
+ * The Card defaults to being "collapsed", the open/closed state is stored in localStorage
  */
 export default function CrashChangeLog({ data }) {
   const [selectedChange, setSelectedChange] = useState(null);
+  const [isOpen, setIsOpen] = useState(
+    localStorage.getItem("showHistory") === "true"
+  );
```

Comment:
this is great! for the item name, would you mind prefixing it with the `appCodeName` like we do elsewhere? and maybe be more specific with the name, like `showCrashHistory`?

Reply:
updated. 
~~every release, the app defaults to not showing the history card until the user clicks it?~~
never mind, I see that the app name needs to be manually updated, and its not pulled from the release notes

I am assuming then that that app name is only updated if there are breaking changes to, say, the grid filters. But that would mean that if there are breaking changes to that config, everything that is stored in local storage gets reset to default, regardless of its affected by other local storage changes?

Reply:
yes, that's right. we only change the app code name if we make a breaking change to storage, and it would have the effect of resetting everyone's storage.